mypyc/ir/ops.py

```diff
@@ -1173,17 +1173,21 @@ def __init__(self,
                  steals: StealsDescription,
                  error_kind: int,
                  line: int,
-                 var_arg_idx: int = -1) -> None:
+                 var_arg_idx: int = -1,
+                 extra_int_constant: Optional[int] = None) -> None:
```

Comment:
It's unnecessary to modify `CallC`. It's better for IR builder to generate a `LoadInt` op for the extra argument.

Reply:
I like this approach!